crates/viewer/re_renderer/src/importer/obj.rs

```diff
@@ -22,6 +25,8 @@ pub enum ObjImportError {
 pub fn load_obj_from_buffer(
     buffer: &[u8],
     ctx: &RenderContext,
+    vertex_colors: &Option<Vec<Color>>,
+    albedo_factor: &Option<AlbedoFactor>,
```

Comment:
I don't think this should be passed into specific mesh format loaders but rather be a universal post processing step.

Which I think is very well illustrated by the fact that this ignored gltf/glb loader